src/Core/Content/Test/ImportExport/Service/ImportExportServiceTest.php

```diff
@@ -241,6 +242,26 @@ public function testExportProfileShouldThrowExceptionInImport($profile, $task, $
         }
     }
 
+    #[DataProvider('profileProvider')]
+    public function testExportProfileShouldNotThrowExceptionWhenExportingErrorLog($profile, $task): void
```

Comment:
Good job :muscle:,
but it would be better if you extend the `profileProvider()` with your test case(s) and just use the `shouldThrowException` parameter to test a "should not throw..." scenario.
Additionally, it might be beneficial to rename the initial test, `testExportProfileShouldThrowExceptionInImport()`, to better align with the "should" and "should not" test cases.

Reply:
Hi @LarsKemper 
sorry, I don't get it.

Since the error log export must not throw an exception, what do you want me to do with the $shouldThrowException parameter?
What name do you suggest for testExportProfileShouldThrowExceptionInImport?

Reply:
I don't like the idea. Your suggestion would combine three different test cases into one, and the name "testProfileExportAndImportPreparation" would cover none of them. 
I'd guess that it's okay to leave these cases separated, even performance-wise. The test only contains a few profiles.